components/dashboard/src/teams/GitIntegrationsPage.tsx

```diff
@@ -4,48 +4,13 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { FunctionComponent } from "react";
-import { Redirect } from "react-router";
-import Header from "../components/Header";
-import { SpinnerLoader } from "../components/Loader";
-import { useCurrentOrg } from "../data/organizations/orgs-query";
 import { GitIntegrations } from "./git-integrations/GitIntegrations";
 import { OrgSettingsPage } from "./OrgSettingsPage";
 
-export default function GitAuth() {
+export default function GitAuthPage() {
     return (
-        <OrgSettingsPageWrapper>
+        <OrgSettingsPage title="Git Auth" subtitle="Configure Git Auth for GitLab or Github.">
             <GitIntegrations />
```

Comment:
**praise**: This is out of the scope of the changes in this PR, but wanted to say thanks for improving the copy in https://github.com/gitpod-io/gitpod/pull/16850 from the previous iteration. 🌮 🌮